charts/umbrella/values.yaml

```
@@ -18,7 +18,7 @@
 # #############################################################################
 ---
 portal:
-  enabled: false
+  enabled: true
```

```suggestion
  enabled: false
```
This is the default values file, containing only the default configuration with all components set to `false`. To deploy with customized values, create a separate file where you enable the desired components by setting them to `true` and override any values you want to change, similar to `values-adopter-data-exchange.yaml`.

---

`charts/umbrella/values-test-data-exchange-azure.yaml` ia now added and default values file is restored.